core-web/libs/portlets/dot-experiments/portlet/src/lib/dot-experiments-configuration/store/dot-experiments-configuration-store.ts

```diff
@@ -38,6 +38,10 @@ import {
     processExperimentConfigProps
 } from '../../shared/dot-experiment.utils';
 
+export const EXP_CONFIG_ERROR_LABEL_CANT_EDIT = 'experiment.configure.edit.only.draft.status';
+
+export const EXP_CONFIG_ERROR_LABEL_PAGE_BLOCKED = 'experiment.configure.edit.page.blocked';
```

Comment:
Move this constant to the constant file.

Reply:
done